### ISSUE TITLE
cannot move in mobile screen

### ISSUE THREAD

**Issue by dieungocbao** [rdbeach/wb#6]
Hello @rdbeach ,
Whiteboard can draw but cannot move the workspace to another position on mobile.
![image](https://user-images.githubusercontent.com/26760639/91677940-6c0de000-eb6e-11ea-8e4f-6731b21d0263.png)

**Comment by scharrua** [comment]
Hi! 
what about using 2 fingers to scroll/move the workspace?

**Comment by rdbeach** [comment]
Use the hand button to move around the screen.

Using two fingers is a good idea. I’ll look into it and see what I can do.

**Comment by dieungocbao** [comment]
> Use the hand button to move around the screen.

When using the hand button, I can't move the toolbar or the workspace.

**Comment by rdbeach** [comment]
Which mobile device are you using? Also which browser? Are you saying that the hand button does not allow you to scroll around the screen?

**Comment by dieungocbao** [comment]
Oh, my bad. I found out that it's working perfectly on physical device (I've test on iPhone with Safari & Chrome) but not working on Android emulator.

**Comment by dieungocbao** [comment]
> Which mobile device are you using? Also which browser? Are you saying that the hand button does not allow you to scroll around the screen?

Yes, I can't move when using hand button. I found out that it's working perfectly on physical device (I've test on iPhone with Safari & Chrome) but not working on Android emulator.

Name: Pixel_2_API_28
Target: Android 9.0

**Comment by itabix** [comment]
Not working on Android (Pixel 3 with Android 11)

**Comment by parkermaths** [comment]
This still appears to be a problem on the current version. It is not just Android, but appears to be other touch enabled devices as well. It did not work on my Surface Pro, Galaxy Tab or Galaxy S9. It also didn't work in the chrome mobile device preview.

The issue appears to be caused by the lines of code below, which are in board.html, just after the body tag:

```
		document.addEventListener('touchmove', function (event) {
  			if (event.scale !== 1) { event.preventDefault(); }
		}, {passive: false} );
```

Removing this line of code appears to resolve the issue for me on all devices. However, I am not sure of the actual purpose of this script, other than it is related to touch events, so I don't know if removing this like will break something else. I haven't found anything that it has broken yet though.

Hope this helps.
